lib/include/openamp/virtio.h

```diff
@@ -565,6 +568,28 @@ static inline int virtio_free_buf(struct virtio_device *vdev, void *buf)
 	return 0;
 }
 
+/**
+ * @brief Check if the virtio device support a specific feature.
+ *
+ * @param vdev		Pointer to device structure.
+ * @param feature_bit	Feature bit to check.
+ *
+ * @return true if the feature is supported, otherwise false.
+ */
+static inline bool virtio_has_feature(struct virtio_device *vdev,
+				      unsigned int feature_bit)
+{
+	uint32_t features;
+
+	if (!vdev)
```

Comment:
We should also check the feature_bit value. For time being we support only up to 32 features bit
so we should check that ` feature_bit <  sizeof(features)`

Reply:
Done